src/sentry.js

```diff
@@ -0,0 +1,16 @@
+import * as Sentry from '@sentry/browser'
```

Comment:
Wondering where we should put this file; I was mainly thinking between this and something like `configuration/sentry.js`.

Reply:
I think we can leave it here for now and if we have the need to move all config files (endpoints, networks, etc) we can do so later?